README.md

```diff
@@ -1,101 +1,171 @@
 
-### Command Line Arguments
+You can mix all three variants if you want:
 
-#### `install`
+    npx elm-test tests "src/**/*Tests.elm" app
 
-Like `elm install`, except it installs to the `test-dependencies` field of your project's `elm.json` file instead of `dependencies`.
+### Inside each file
 
-```shell
-elm-test install elm/regex
+For elm-test to find tests in your files you need to:
+
+1. Create top-level values of the type [Test](https://package.elm-lang.org/packages/elm-explorations/test/latest/Test#Test). You can name the values anything – the only thing that matters is that their type is `Test`.
+2. Expose them.
+
+Example:
+
+```elm
+module LoginForm exposing (alreadyLoggedInTests, tests)
+
+import Test exposing (Test)
+
+
+tests : Test
+tests =
+    -- ...
+
+
+alreadyLoggedInTests : Test
+alreadyLoggedInTests =
+    -- ...
 ```
 
-#### `init`
+Some prefer to expose a single `Test` value and group everything using [describe](https://package.elm-lang.org/packages/elm-explorations/test/latest/Test#describe). Some prefer to expose several `Test` values.
+
+**Also check out the [elm-explorations/test quick-start](https://github.com/elm-explorations/test#quick-start)!**
+
+## Command Line Arguments
+
+This is the most common commands and flags. Run `elm-test --help` for an exhaustive list.
+
+**Note:** Throughout this section, the `npx` prefix is omitted for brevity.
+
+### install
 
-Runs `elm-test install elm-explorations/test` and then creates a `tests/Example.elm`
-example test to get you started.
+Like `elm install`, except it installs to `"test-dependencies"` in your `elm.json` instead of to `"dependencies"`.
+
+    elm-test install elm/regex
+
+### init
+
+Runs `elm-test install elm-explorations/test` and then creates a `tests/Example.elm` example test to get you started.
+
+An `elm.json` is required, so you need to run `elm init` first if you don’t already have one.
 
 Afterwards, you can run `elm-test` with no arguments to try out the example.
 
-#### `--compiler`
+    elm init
+    elm-test init
+    elm-test
 
-The `--compiler` flag can be used to use a version of the Elm compiler that
-has not been installed globally.
+### --watch
 
-```shell
-npm install elm
-elm-test --compiler ./node_modules/.bin/elm
-```
+Starts the runner in watch mode. Your tests will automatically rerun whenever any test file or a file they import are changed.
 
-#### `--seed`
+    elm-test --watch
 
-Allow running the tests with a predefined seed, rather than a randomly generated seed. This is especially helpful when trying to reproduce a failing fuzz-test.
+### --seed
 
-```shell
-elm-test --seed=12345
-```
+Run with a previous fuzzer seed, rather than a randomly generated seed. This allows reproducing a failing fuzz-test. The command needed to reproduce (including the `--seed` flag) is printed after each test run. Copy, paste and run it!
```

Comment:
Is there a use case for running with a specific seed other than a previous one?

Reply:
I cannot think of any use case. But it would work if you wanted it to. Maybe in CI you want to always run with the same seed to avoid spurious failures?

Reply:
Makes sense. I’m changing to “specific”.